src/components/Layout/Contact.tsx

```diff
@@ -13,73 +13,145 @@ import {
   Text,
   Grid,
   GridItem,
-  Textarea
+  Textarea,
+  useToast
 } from "@chakra-ui/react";
-
+import { sendEmail } from "../../utils/fetchData";
+import { error } from "console";
 const Contact = () => {
+  const toast = useToast();
+  const handleSendEmail = (e: any) => {
+    e.preventDefault();
+    const data = new FormData(e.target);
+    const dataObject = Object.fromEntries(data.entries());
+    sendEmail(dataObject)
+      .then(response => {
+        console.log(response);
```

Comment:
BLOCKING: You probably want to remove this `console.log` since you only need it when developing locally.